Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CylindricalYeeAlgorithm.H

```diff
@@ -42,6 +43,7 @@ struct CylindricalYeeAlgorithm {
      */
     static amrex::Real ComputeMaxDt ( amrex::Real const * const dx,
                                       int const n_rz_azimuthal_modes ) {
+        using namespace amrex::literals;
```

Comment:
I think this is the only line where you need `literals` explicitly.

`amrex::literals` is an "inline" namespace of `amrex` and always pulled otherwise if you already do `using namespace amrex;`

Reply:
update: and `ComputeMaxDt`